### DISCUSSION TITLE
Beberapa Plot Tidak Mau Tampil (Capstone DV)

### DISCUSSION THREAD

**Discussion by RamadhanRC** [teamalgoritma/community#424] [Q&A] [answered]
Beberapa Plot saya tidak mau tampil saat di Run App maupun saat sudah di publish di Shinyapps.io mohon bantuannya

ini link shinyapps.io
https://ramadhanrc.shinyapps.io/CapstoneDV_Company_Salaries

ini logsnya
[logs.txt](https://github.com/teamalgoritma/community/files/9893713/logs.txt)

**Comment by RamadhanRC** [comment]
[logs (4).txt](https://github.com/teamalgoritma/community/files/9893870/logs.4.txt)


Update logs, sepertinya sudah tidak ada error lagi, tapi dari 6 plot masih hanya 2 plot saja yg muncul di shinyapps.io jadi mohon bantuannya ya

**Comment by kinantytasyao** [comment, marked as answer]
Dear Pak @RamadhanRC 

Apabila ketika di run app tidak muncul kemungkinan ada kesalahan code, coba untuk dipastikan kembali apakah terdapat kesalahan pada codenya. Jika memang masih tidak bisa mungkin dapat melampirkan zipnya karena kami tidak bisa melihat debugnya dari logs.

Demikian yang dapat kami sampaikan. Terima kasih dan semoga membantu.

**Reply by RamadhanRC** [reply]
@kinantytasyao berikut saya lampirkan link gdrive data saya yang sudah di ZIP https://drive.google.com/drive/folders/1i2CdYtSyZpd3Hq8-dxEA-cQ5nFo3ShVV?usp=share_link

**Comment by RamadhanRC** [comment]
Aduh Algoritma Community ini ga membantu ya!!!! Ga pernah Solved kalo nanya disini :(

**Reply by kinantytasyao** [reply]
Pada file ui.R untuk plot_5 di line 110 ganti plotlyOutput menjadi highchartOutput karena Bapak menggunakan plot dari package highchart dan untuk plot_6 di line di line 59 ganti plotlyOutput menjadi leafletOutput karena Bapak menggunakan plot dari package leaflet.

Demikian yang dapat kami sampaikan. Terima kasih.
